wagtail/admin/views/pages/delete.py

```diff
@@ -50,5 +78,10 @@ def delete(request, page_id):
             "page": page,
             "descendant_count": page.get_descendant_count(),
             "next": next_url,
+            "translation_count": len(pages_to_delete),
+            "translation_descendant_count": sum(
+                [p.get_descendants().count() for p in pages_to_delete]
+            ),
+            "combined_subpages": page.get_descendant_count() + len(pages_to_delete),
```

Comment:
ditto for combined subpages. we're only talking about translated (ie. non-alias) pages